hpat/tests/test_series.py

```diff
@@ -72,6 +71,19 @@
     '1234567890',
 ]
 
+def gen_srand_array(size, nchars=8):
+    """Generate array of strings of specified size based on [a-zA-Z] + [0-9]"""
+    accepted_chars = list(string.ascii_letters + string.digits)
+    rands_chars = np.array(accepted_chars, dtype=(np.str_, 1))
+
+    np.random.seed(100)
+    return np.random.choice(rands_chars, size=nchars * size).view((np.str_, nchars))
+
+
+def gen_frand_array(size, min=-100, max=100):
+    """Generate array of float of specified size based on [-100-100]"""
+    float_list = (max - min) * np.random.sample(size) + min
+    return float_list
```

Comment:
Why not `return (max - min) * np.random.sample(size) + min`?

Reply:
@densmirn I have, at least, one explanation - it is easier to debug return value if you have it in separate value. You an not see result of equations if it written in `return` operator.
So, I would say this is a good practice to have `float_list` and use it in return later.

Reply:
Actually you are always able to debug the result outside of the function.